### ISSUE TITLE
Draft Notify TOS

### ISSUE THREAD

**Issue by tdlowden** [GSA/notifications-admin#348]
- [ ]  Search for other TTS TOSes
- [ ] Refer to Login's (sent by Dan Nicotera) and Search.gov (on their website)
- [ ] Create draft TOS
- [ ] Get team review
- [ ] Get OGC Tech Law Div review

**Comment by tdlowden** [comment]
Do we think this needs to be done prior to pilot?

**Comment by tdlowden** [comment]
Q to bring to OGC: Is this needed for pilot partners, even though they are signing an MOU?

**Comment by tdlowden** [comment]
I think we have determined that the Terms are outlined in the MOUs themselves and unless we move to a model without MOUs/IAAs, this is not needed for the site
